config/sync/views.view.entity_clone_template.yml

```diff
@@ -812,6 +861,55 @@ display:
           hide_empty: false
           empty_zero: false
           hide_alter_empty: false
+        nothing_1:
+          id: nothing_1
+          table: views
+          field: nothing
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: custom
+          label: 'Edit template link'
```

Comment:
Please consider removing (or hiding) the label.

It stands out somewhat in the provided screen shot. We do not show labels for the other fields.